app/src/main/java/com/realityexpander/tasky/agenda_feature/data/repositories/eventRepository/eventRepositoryImpls/EventRepositoryImpl.kt

```diff
@@ -1,4 +1,4 @@
-package com.realityexpander.tasky.agenda_feature.data.repositories.eventRepository.eventRepositoryImpls
+package com.realityexpander.eventy.agenda_feature.data.repositories.eventRepository.eventRepositoryImpls
 
```

Comment:
Did you rename the app to Eventy?

Or did you rename Task to Event and it got renamed by mistake?

Reply:
Whoops, I will fix it.